docs/troubleshooting.md

```diff
@@ -456,7 +456,23 @@ Address: 50.17.235.25
 
 ### Elasticsearch crashloop under EKS
 
-EKS clusters which use the AMI image named `amazon-eks-node-1.10-v20190220` in any nodegroup are known to be affected by a [bug](https://github.com/awslabs/amazon-eks-ami/issues/193) that causes Elasticsearch 6.0 to get into a crashloop. In order to fix this, you will SSH into every node from the EKS nodegroup and remove the `ulimit` configuration of the Docker daemon:
+When Elasticsearch enters into a never-ending crashloop under EKS, the logs can reveal the cause:
+
+```console
+$ kubectl --namespace=kubeprod logs elasticsearch-logging-0 elasticsearch-logging
+```
+
+__Troubleshooting__:
+
+If you see logged mesasges like this:
+
+```
+[1]: max file descriptors [8192] for elasticsearch process is too low, increase to at least [65536]
+```
+
+Then, you are affected by a [bug](https://github.com/awslabs/amazon-eks-ami/issues/193) that causes Elasticsearch 6.0 to get into a crashloop because of incorrect `ulimit` values. It affects EKS clusters which use the AMI image named `amazon-eks-node-1.10-v20190220`.
+
+In order to fix this, you will SSH into every node from the EKS nodegroup and remove the `ulimit` configuration of the Docker daemon:
```

Comment:
s/you will SSH into every node/you will need to SSH into every node/

Reply:
Fixed.